projects/plugins/jetpack/extensions/plugins/payments/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { ExternalLink } from '@wordpress/components';
+import { PluginPostPublishPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import analytics from '../../../_inc/client/lib/analytics';
+import useAutosaveAndRedirect from '../../shared/use-autosave-and-redirect/index';
+
+const PaymentsPostPublish = () => {
+	const paymentInfoUrl = 'https://wordpress.com/payments-donations/';
+	const { autosaveAndRedirect } = useAutosaveAndRedirect( paymentInfoUrl );
+	const trackClick = event => {
+		event.preventDefault();
+		analytics.tracks.recordEvent( 'jetpack_editor_payments_post_publish_click' );
+		autosaveAndRedirect( event );
+	};
+
+	return (
+		<PluginPostPublishPanel
+			className="jetpack-payments-post-publish-panel"
+			title={ __( 'Start accepting payments', 'jetpack' ) }
+			initialOpen
+		>
+			<p>
+				{ __(
+					'Insert the Payments block button or the Donations block form -- no plugin required.',
```

Comment:
`--` should probably be `—`, but I'll defer to a real editorial review. 😄

Reply:
Good point. 6049599541c6d8cadc99e83bc7bac7bc6f381676